src/components/calcite-block/calcite-block.e2e.ts

```diff
@@ -104,24 +104,47 @@ describe("calcite-block", () => {
       expect(summary.innerText).toBe("test-summary");
     });
 
-    it("supports a nested control", async () => {
-      const page = await newE2EPage();
-      await page.setContent(
-        `<calcite-block heading="test-heading" collapsible><input class="nested-control" slot=${SLOTS.control} /></calcite-block>`
-      );
-      const element = await page.find("calcite-block");
-      const elementToggleSpy = await element.spyOnEvent("calciteBlockToggle");
-
-      const control = await element.find(".nested-control");
-      expect(await control.isVisible()).toBe(true);
-      expect(await control.getProperty("slot")).toEqual(SLOTS.control);
-
-      await control.click();
-      expect(elementToggleSpy).toHaveReceivedEventTimes(0);
-
-      await element.click();
-      await element.click();
-      expect(elementToggleSpy).toHaveReceivedEventTimes(2);
+    describe("adding a header control", () => {
+      it("allows users to add a control in a collapsible block", async () => {
```

Comment:
Seems you're testing multiple things inside this one test block. Consider breaking this into separate isolated tests so it's clear what's working vs. what's breaking when this test passes/fails.

Reply:
There are multiple assertions, but they're all related to the same test.